src/fluree/server/system.clj

```diff
@@ -73,23 +97,16 @@
   (jetty/stop-server http-server))
 
 (defn start-config
-  ([config]
-   (start-config config {}))
-  ([config overrides]
-   (-> config
-       (meta-merge overrides)
-       ig/init)))
+  [config]
+  (-> config ig/expand ig/init))
 
 (defn start-resource
```

Comment:
I've got a general question about the config files - if they are `resource`s, how will users who are just working with our jar artifact change them? My understanding (which may be wrong) is that everything in the `resource` dir gets bundled into the jar, and I don't see us looking anywhere else for a config file.

Reply:
That's right. The resource files are the baseline defaults in my mind. You can override those defaults with environment variables only right now, but I think we can add functionality later that allows the user to load a config file from an absolute path on the system pretty easily if we want to do that.

Reply:
I think we may need it sooner than later, @aaj3f was talking about using config files on the docker images, but thanks for confirming my understanding :+1:

Reply:
Ok. I added some command line options parsing and the ability to specify a config file path on the file system to load.